.buildkite/pipeline.yml

```
@@ -133,31 +133,17 @@ steps:
     commands:
     - nix-build ci.nix -A haddock --no-out-link
 
-  # TODO #124 uncomment:
-  # - label: scheduled florencenet ligo test
-  #   if: build.source == "schedule"
-  #   env:
-  #     TASTY_NETTEST_NODE_ENDPOINT: "http://florence.testnet.tezos.serokell.team:8732"
-  #   depends_on:
-  #     - build-ligo
-  #     - build-haskell
-  #   commands: *ligo-nettest
-  #   retry:
-  #     automatic:
-  #       limit: 1
-
-  # TODO #124 uncomment:
-  # - label: scheduled granadanet ligo test
-  #   if: build.source == "schedule"
-  #   env:
-  #     TASTY_NETTEST_NODE_ENDPOINT: "http://granada.testnet.tezos.serokell.team:8732"
-  #   depends_on:
-  #     - build-ligo
-  #     - build-haskell
-  #   commands: *ligo-nettest
-  #   retry:
-  #     automatic:
-  #       limit: 1
+  - label: scheduled granadanet ligo test
+    if: build.source == "schedule"
```

I suggest making this change temporarily:
```suggestion
    if: *not_scheduled
```
to test it in this PR (and revert it before merging)